client/src/app/pages/stock-detail/stock-detail.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container" fxLayout="row wrap" fxLayoutGap="16px grid">
-    <app-stock-detail-header [stockInfo]="stockInfo$ | async" fxFlex="67%" fxFlex.lt-sm="100%" ></app-stock-detail-header>
-    <div fxFlex="33%" fxFlex.lt-sm="100%" >side component</div>
+    <app-stock-detail-header [stockInfo]="stockInfo$ | async" fxFlex="65%" fxFlex.lt-sm="100%" ></app-stock-detail-header>
+    <div fxFlex="35%" fxFlex.lt-sm="100%" ><app-stock-stats [stockInfo]="stockInfo$ | async"></app-stock-stats></div>
```

Comment:
[Question] Is there a reason these values were changed? 67% -> 65%, 33% -> 35%

Reply:
yes I just need a bit more width for the stats since few properties are big numbers which can affect the stats row display

Reply:
ok perfect, then we can keep it this way